hal/src/emmc.c

```diff
@@ -1,50 +1,225 @@
 #include <hal/emmc.h>
-#include <drivers/sdmmc/fsl_mmc.h>
-#include <drivers/sdmmc/fsl_sdmmc_host.h>
+#include <boot/board.h>
+#include "MIMXRT1051.h"
 #include <errno.h>
 #include <stdbool.h>
 
+#include <drivers/fsl_iomuxc.h>
+#include <hal/delay.h>
+
+//#define DEBUG_USDHC
```

Comment:
I'd add some comment here about what happens after un-commenting it so that nobody will remove it in the future while cleaning up the code 😉

Reply:
I decided to remove the debug code for USDHC in recovery. It is not necessary and I left this code only in ecoboot ;)